modules/ossm-kiali-about.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/kiali/ossm-kiali-assembly.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-kiali-about_{context}"]
+= About Kiali
+:context: ossm-kiali-about
+
+// there is only 1 attribute for Kiali at this time
+{KialiProduct} lets you view configurations, monitor traffic, and analyze traces in a single console. It is based on the open source link:https://www.kiali.io/[Kiali] project.
+
+Kiali is the management console for {SMProductName}. It provides dashboards, observability, and robust configuration and validation capabilities. It shows the structure of your service mesh by inferring traffic topology and displays the health of your mesh. Kiali provides detailed metrics, powerful validation, access to Grafana, and strong integration with the {TempoName}.
```

Comment:
This isn't specific feedback but just wanted to point this out - if "Kiali" at the beginning here is meant to reference the Operator and not the open source project, I think this assembly has an overall clarity issue between Kiali (the project) and Kiali (the shortened Operator name), though I'm not sure how that would best be addressed.